content/F2015/F15-event-dropbox-coffee-chats.md

```diff
@@ -0,0 +1,18 @@
+title: Dropbox Coffee Chats
+Date: 2015-09-05
+Category: Events
+Tags: social, industry
+Slug: dropbox-coffee-chats-f15
+Author: Anna Lorimer
+Summary: Come and grab some coffee and snacks and chat with some female engineers from Dropbox!
+
+Join us and several female engineers from Dropbox for coffee and snacks at William's in the plaza.
+
+Note: This is a [women-only](http://wics.uwaterloo.ca/faq/) event.
```

Comment:
I wouldn't say we _just_ started doing it. I try to point it out when I see it, but it's not the worst thing to have on. Relative links are nice in case we change our base url at some point. I think it would be good practice to keep doing it for new pages as we make them but if someone is vehemently opposed to it it's not super important to me.